src/templates/avatar.ts

```diff
@@ -1,61 +1,49 @@
 import {createHash} from 'crypto';
-import Handlebars from 'handlebars';
+import {html, safe} from '../types/html';
 
 function getGravatarUrl(email: string, size: number = 160) {
   const trimmedEmail = email.trim().toLowerCase();
   const hash = createHash('sha256').update(trimmedEmail).digest('hex');
   return `https://www.gravatar.com/avatar/${hash}?s=${size}&d=identicon`;
 }
 
-const AVATAR_THUMBNAIL_TEMPLATE = Handlebars.compile(
-  `
-    <img
-      width="40"
-      height="40"
-      srcset="{{url1x}} 1x, {{url2x}} 2x, {{url4x}} 4x"
-      src="{{url1x}}"
-      alt="The avatar of {{memberNumber}}"
-    />
-  `
-);
+type GravatarViewModel = {
+  url1x: string;
+  url2x: string;
+  url4x: string;
+  memberNumber: number;
+};
 
-const AVATAR_PROFILE_TEMPLATE = Handlebars.compile(
-  `
+const gravatar =
+  (width: number, height: number) => (viewModel: GravatarViewModel) => html`
     <img
-      width="320"
-      height="320"
-      srcset="{{url1x}} 1x, {{url2x}} 2x, {{url4x}} 4x"
-      src="{{url1x}}"
-      alt="The avatar of {{memberNumber}}"
+      width="${width}"
+      height="${height}"
+      srcset="
+        ${safe(viewModel.url1x)} 1x,
+        ${safe(viewModel.url2x)} 2x,
+        ${safe(viewModel.url4x)} 4x
```

Comment:
Should I push this safe assert up to when the urls are generated - I think probably yes